tests/ga/test_extension.py

```diff
@@ -2834,6 +2841,19 @@ def test_success_code_should_set_in_env_variables_on_cmd_success(self, _):
             self.assertTrue(test_env_file_name + " -u" in update_kwargs['message'] and "%s=%s" % (
                 ExtCommandEnvVariable.DisableReturnCode, exit_code) in update_kwargs['message'])
 
+    @patch('time.sleep', side_effect=lambda _: mock_sleep(0.0001))
+    def test_disable_should_not_be_called_on_disabled_ext_during_version_upgrade(self, _):
```

Comment:
NIT: The test name is a bit confusing, it should be more specific, something like - 
`test_disable_should_not_be_called_during_version_upgrade_if_not_enabled`